### PR TITLE
Cosmos example

### PR DESCRIPTION
**Pull request by patiee**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -10,8 +10,12 @@
     "deploy-local": "graph deploy cosmos-example -l v0.1.0 --ipfs http://localhost:5001 --node http://localhost:8020",
     "remove-local": "graph remove cosmos-example --node http://localhost:8020"
   },
-  "devDependencies": {},
+  "devDependencies": {
+    "@graphprotocol/graph-cli": "file:../graph-cli",
+    "@graphprotocol/graph-ts": "file:../graph-ts"
```
Comment (lavabyrd) [inline package.json:15]: local testing?

Comment (patiee) [reply package.json:15]: yes local testing, I'll update it now

File: package.json [hunk 2]
```diff
@@ -10,8 +10,12 @@
     "deploy-local": "graph deploy cosmos-example -l v0.1.0 --ipfs http://localhost:5001 --node http://localhost:8020",
     "remove-local": "graph remove cosmos-example --node http://localhost:8020"
   },
-  "devDependencies": {},
+  "devDependencies": {
+    "@graphprotocol/graph-cli": "https://github.com/figment-networks/graph-cli.git#tendermint",
+    "@graphprotocol/graph-ts": "https://github.com/figment-networks/graph-ts.git#patiee/cosmos-txs"
+  },
   "dependencies": {
+    "as-proto": "https://github.com/figment-networks/as-proto.git",
```
Comment (lavabyrd) [inline package.json:18]: can we version lock this if possible?

**Comment by lavabyrd** [comment]
looks like we've a yarn.lock and a package lock file, do we need both?

File: package.json (hunk 2 above)
Comment (patiee) [reply package.json:18]: should I create a new tag on that branch?

Comment (lavabyrd) [reply package.json:18]: yup!

**Pull request merged**